.github/workflows/mr_ci.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest]
-        python-version: [3.8]
+        python-version: [3.9, 3.12]
```

Comment:
I assume that previously it was supposed to run on 3.8+ and not exclusively 3.8, but that the CI/testing just wasn't checking them all? I guess you could add 3.10 and 3.11 too, but the tradeoff might be that merging to main takes ages, depends how often you do that I guess

Reply:
I am reluctant to add too many python versions to the test as they take quite a while. 
The idea behind 3.9 and 3.12 here is lower limit and upper limit and so I was hoping this would catch any issues.